content/applications/marketing/social_marketing/essentials/social_campaigns.rst

```diff
@@ -0,0 +1,139 @@
+==========================
+Social marketing campaigns
+==========================
+
+Social marketing campaigns help companies connect directly with the marketplace. These campaigns
+are vital when introducing a new product to customers, explaining the value of a service/product,
+or advertising an upcoming event or product release.
+
+The most effective social marketing campaigns involve multiple channels, so it's important to have
+a reliable solution to help with the planning, posting, tracking, and analyzing of content.
+
+Campaigns page
+==============
+
+To access a complete overview of all social marketing campaigns, open the :menuselection:`Social
+Marketing` application, and click :menuselection:`Campaigns` from the header menu. Doing so reveals
+a separate page with every campaign in a default kanban view.
+
+.. image:: social_campaigns/campaigns-page.png
+   :align: center
+   :alt: View of the campaigns page in the Odoo Social Marketing application.
+
+Each *stage* in the kanban view can be edited, by clicking the "gear" icon to the left of the "+"
+(plus sign) - located to the right of the stage title.
+
+The "gear" icon only appears when the cursor hovers to the left of the "+" (plus sign). When the
+"gear" icon is clicked, a drop-down menu reveals the options: *Fold*, *Edit Stage*, and *Delete*.
+
+.. image:: social_campaigns/campaign-stage-dropdown.png
+   :align: center
+   :alt: View of the campaigns page in the Odoo Social Marketing application.
+
+Clicking *Fold* minimizes that specific stage's column. The stage column can be restored by clicking
+the folded version of it on the main *Campaigns* page in the default kanban view.
+
+Selecting *Edit Stage* reveals a pop-up, in which the name and the sequence of the stage can be
+modified. If changes are made, be sure to click :guilabel:`Save`.
+
+Clicking *Delete* removes the stage entirely.
```

Comment:
Since we're actively on this dashboard and providing a UI walk-through, these get guilabels.

```suggestion
Clicking :guilabel:`Fold` minimizes that specific stage's column. The stage column can be restored
by clicking the folded version of it on the main :guilabel:`Campaigns` dashboard in the default
kanban view.

Selecting :guilabel:`Edit Stage` reveals a pop-up window, in which the name and the sequence of the
stage can be modified. If changes are made, be sure to click :guilabel:`Save`.

Clicking :guilabel:`Delete` removes the stage entirely.
```

cc: @samueljlieber